Assignments/Assignment2.java

```diff
@@ -0,0 +1,33 @@
+package Assignments;
+
+import java.util.Scanner;
+
+public class Assignment2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int count[] = new int[26];
+        int counter = 0;
+        boolean flag = false;
+        //String str = sc.nextLine();
+        String str = "The quick brown fox jumps over the lazy dog";
+        for (int i = 0; i < str.length(); i++) {
```

Comment:
Can't you split this into method(string)??
Reason:
    The method could be reused if required but  not the for loop